tfx/components/transform/executor.py

```diff
@@ -536,6 +558,7 @@ class _ToArrowRecordBatchesFn(beam.DoFn):
     """Converts a batch of serialized examples to an Arrow RecordBatch."""
 
     def __init__(self, schema: Optional[schema_pb2.Schema]):
+      super().__init__()
```

Comment:
is there an issue before adding those? let's keep the change minimum otherwise there will be a lot of conflict. refactor can be done in separate PRs if needed

Reply:
yes, the pylint test failed before I added this, the same pylint issue like tfdv

Reply:
let's drop this, internal lint doesn't check this